pkg/api/vlabs/types.go

```diff
@@ -38,6 +38,7 @@ type Properties struct {
 	MasterProfile           MasterProfile           `json:"masterProfile"`
 	AgentPoolProfiles       []AgentPoolProfile      `json:"agentPoolProfiles"`
 	LinuxProfile            LinuxProfile            `json:"linuxProfile"`
+	Environment				Environment				`json:"environment"`
```

Comment:
Thanks for reviewing it. I am on vacation for Chinese new year and I will send the update once I get back to the office.